.gitignore

```diff
@@ -1,3 +1,4 @@
 Manifest
 pkg
 output
+.idea
```

Comment:
if you put that in your global .gitignore, you never have to add it to any project :)
https://github.com/grosser/dotfiles/blob/master/gitconfig#L66-L67